src/components/Notifications/Form/IntegrationRecipientTypeahead.tsx

```diff
@@ -19,6 +19,7 @@ export interface IntegrationRecipientTypeaheadProps extends OuiaComponentProps {
     isDisabled?: boolean;
     onSelected: (recipientOption: RecipientOption) => void;
     onOpenChange?: (isOpen: boolean) => void;
+    validated: string | undefined;
```

Comment:
If following the above suggestion of using a boolean, this would change to
```suggestion
    validated: boolean;
```